bundles/org.openhab.core/src/main/java/org/openhab/core/events/system/SystemEventFactory.java

```diff
@@ -19,7 +19,6 @@
 import org.openhab.core.events.AbstractEventFactory;
 import org.openhab.core.events.Event;
 import org.openhab.core.events.EventFactory;
-import org.openhab.core.types.Type;
```

Comment:
When removing it here, you probably need to fully qualify it in the Javadoc (e.g. line 40).

Reply:
Done